tspng/test_extraction.py

```diff
@@ -22,6 +22,12 @@ def test_extract():
     test_data = extract_from_folder('tests/assets')
     assert list(test_data.keys())==['tests/assets/example_file_1.ts.png','tests/assets/example_file_2.ts.png']
 
+def test_extract_fails():
+    try:
```

Comment:
This is not correct. The `with pytest.raises(TypeError):` structure should be used.

```python
def test_extract_fails():
    with pytest.raises(TypeError):
        extract("A string")
```

This style should be used for all of the "failing" tests.

Reply:
Will make this change for those tests